os/test/src/PathTests.scala

```diff
@@ -436,5 +439,34 @@ object PathTests extends TestSuite {
       assert(result1 == expected)
       assert(result2 == expected)
     }
+    test("issue201") {
+      Path("/omg") // rootRelative path does not throw exception.
+    }
+  }
+  // compare absolute paths
+  def sameFile(a: java.nio.file.Path, b: java.nio.file.Path): Boolean = {
+    a.toAbsolutePath == b.toAbsolutePath
+  }
+  def sameFile(a: os.Path, b: java.nio.file.Path): Boolean = {
+    sameFile(a.wrapped, b)
+  }
+  def sameFile(a: Path, b: Path): Boolean = {
+    sameFile(a.wrapped, b.wrapped)
+  }
+  implicit class ExtendString(s: String) {
+    def posix: String = s.replace('\\', '/')
+    def norm: String = if (s.startsWith(platformPrefix)) {
+      s.posix // already has platformPrefix
+    } else {
+      s"$platformPrefix${s.posix}"
+    }
+  }
+  implicit class ExtendOsPath(p: os.Path) {
+    def posix: String = p.toNIO.toString.posix
+    def norm: String = p.toString.norm
+  }
+  implicit class ExtendPath(p: java.nio.file.Path) {
+    def posix: String = p.toString.posix
+    def norm: String = p.toString.norm
```

Comment:
Let's use normal helper methods here, rather than extension methods, unless we have good reason to do otherwise.

Reply:
done (will push changes soon)